dash_docs/chapters/dash_enterprise/application_structure/index.py

```diff
@@ -0,0 +1,133 @@
+# -*- coding: utf-8 -*-
+import dash_html_components as html
+from dash_docs import reusable_components as rc
+import dash_core_components as dcc
+from dash_docs import tools
+import os
+
+content = tools.load_markdown_files(__file__)
+check_url = tools.is_in_dash_enterprise()
+
+PAGE_CONTENT_PY = [rc.Markdown('''
+
+{buildpack}
+{project_folder_py}
+{lifecycle_py}
+# Required Files
+{app_py}
+{requirements_py}
+{procfile_py}
+# Optional Files
+{checks}
+{app_json}
+{dokku_scale}            
+{gitignore}
+{runtime_py}
+'''.format(**{k.replace('.md', ''): v for (k, v) in content.items()}).format(
+kubernetes_notes = (
+'' if check_url else '''
+> **To view the Kubernetes docs**, visit: https://<your-dash-enterprise-hostname\>/Docs/Kubernetes, 
```

Comment:
```suggestion
> **To view the Kubernetes docs**, visit: https://<your-dash-enterprise-hostname\>/Docs/kubernetes, 
```

Also double check that this typo wasn't made anywhere else.